app/app.py

```diff
@@ -68,7 +105,10 @@ def __call__(self, environ, start_response):
 # Metadata
 metadata_dataframe = pd.read_csv(metadata_file, sep='\t')
 metadata_dataframe['sample_name'] = metadata_dataframe['Sample']
-# metadata_dataframe['sample_name'] = ['sample_'+str(x).replace('.', '_') for x in metadata_dataframe['Sample']]
+# Have the same label names as logcpm columns.
+metadata_dataframe['sample_name'] = ['sample_'+str(x).replace('.', '_') for x in metadata_dataframe['Sample']]
```

Comment:
Instead of doing this which is only applicable to the specific data files you're working with, I think it's better to just augment the data to be properly formatted where the metadata and data indices match.